Embedded Java Engine Samples/Basic/Data Source Usage/BasicMysql/BasicMySql/WindwardReports.properties

```diff
@@ -0,0 +1,224 @@
+
+# You need to enter your license on the line below. It is a uuencoded string and can be in a single line or in
+# multiple lines with each line (except the last one) ending with a \.
+license=[[LICENSE]]
```

Comment:
Remove [[LICENSE]] so they can run it without getting an error right away.  This should be done for each project, just leaving a single comment here about it